public/pages/DetectorResults/containers/AnomalyHistory.tsx

```diff
@@ -514,93 +529,106 @@ export const AnomalyHistory = (props: AnomalyHistoryProps) => {
-                dateRange={zoomRange}
-                featureDataSeriesName="Feature output"
-                showFeatureMissingDataPointAnnotation={
-                  props.detector.enabled &&
-                  // disable showing missing feature alert when it is HC Detector
-                  !isHCDetector
-                }
-                isFeatureDataMissing={props.isFeatureDataMissing}
-                isHCDetector={isHCDetector}
-                selectedHeatmapCell={selectedHeatmapCell}
-              />
-            ) : (
-              [
-                isHCDetector
-                  ? [
-                      <AnomalyOccurrenceChart
-                        title={
-                          selectedHeatmapCell
-                            ? selectedHeatmapCell.entityValue
-                            : '-'
-                        }
-                        dateRange={dateRange}
-                        onDateRangeChange={handleDateRangeChange}
-                        onZoomRangeChange={handleZoomChange}
-                        anomalies={
-                          anomalyResults ? anomalyResults.anomalies : []
-                        }
-                        bucketizedAnomalies={
-                          bucketizedAnomalyResults !== undefined
-                        }
-                        anomalySummary={bucketizedAnomalySummary}
-                        isLoading={isLoading || isLoadingAnomalyResults}
-                        anomalyGradeSeriesName="Anomaly grade"
-                        confidenceSeriesName="Confidence"
-                        showAlerts={true}
-                        isNotSample={true}
-                        detector={props.detector}
-                        monitor={props.monitor}
-                        isHCDetector={isHCDetector}
-                        selectedHeatmapCell={selectedHeatmapCell}
-                      />,
-                      <EuiSpacer size="m" />,
-                    ]
-                  : null,
-                <AnomalyResultsTable
-                  anomalies={filterWithHeatmapFilter(
-                    bucketizedAnomalyResults === undefined
-                      ? anomalyResults
-                        ? filterWithDateRange(
-                            anomalyResults.anomalies,
-                            zoomRange,
-                            'plotTime'
-                          )
-                        : []
-                      : pureAnomalies,
-                    selectedHeatmapCell,
-                    true,
-                    'plotTime'
-                  )}
+        <div style={{ padding: '20px' }}>
+          {isHCDetector
+            ? [
+                <AnomalyOccurrenceChart
+                  title={
+                    selectedHeatmapCell && props.detector?.categoryField ? (
+                      <EuiTitle size="s">
+                        <div>
+                          {`${get(props, 'detector.categoryField.0')}`}{' '}
```

Comment:
Add some TODO for multi-category field to make it easier to track when develop multi-category feature?

Reply:
Sure, I can add. The charts themselves will likely need some changes to support this as well.